src/App.scss

```
@@ -0,0 +1,20 @@
+.App {
+  margin: 30px;
+}
+
+.button {
+  margin: 20px 10px 10px 0;
+}
+
+.box {
+  width: 490px;
+
+  cursor: pointer;
+  transition: background-color, transform 0.3s;
```

If you want to set duration for certain CSS properties, you should write duration time for each one.
```suggestion
  transition: background-color 0.3s, transform 0.3s;
```

---

In your case `background-color` transition duration will be 0sec by default